dev/devicelab/lib/tasks/perf_tests.dart

```diff
@@ -1459,8 +1434,49 @@ class CompileTest {
         throw Exception('Unsupported option for fake devices');
       case DeviceOperatingSystem.fuchsia:
         throw Exception('Unsupported option for Fuchsia devices');
+      case DeviceOperatingSystem.ios:
```

Comment:
Instead of moving the iOS logic down, instead move the `case DeviceOperatingSystem.macos:` logic up to make git spelunking easier.

Reply:
Does it not matter that the switch cases are alphabetized?

Reply:
Nah.